src/panel.js

```diff
@@ -70,7 +82,11 @@ chrome.storage.sync.get({ defaultOpenDepth: 2 }, (items) => {
             }
             if (request.request.headers.find((header) => header.name === 'x-inertia') ||
                 request.response.headers.find((header) => header.name === 'X-Inertia')) {
-                request.getContent(renderJson)
+                if (request.request.headers.some((header) => header.name === 'x-inertia-partial-data')) {
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**🔨 issue:** ‏Header name can also have a title case with older version of Inertia `X-Inertia-Partial-Data`

Here I made a clearer solution for that
```javascript
            const inertiaHeaders = ['x-inertia', 'X-Inertia']
            const inertiaPartialHeaders = ['x-inertia-partial-data', 'X-Inertia-Partial-Data']

            if (request.request.headers.some((header) => inertiaHeaders.includes(header.name))) {
                const isPartial = request.request.headers.some((header) => inertiaPartialHeaders.includes(header.name))
                request.getContent((content) => renderJson(content, isPartial))
            }
```

Reply:
Another way to do it is simply comparing lowercase names 
```js
if (request.request.headers.some((header) => header.name.toLowerCase() === 'x-inertia')) {
    const isPartial = request.request.headers.some((header) => header.name.toLowerCase() === 'x-inertia-partial-data')
    request.getContent((content) => renderJson(content, isPartial))
}
```

Reply:
Done ! I have limited the check of `X-Inertia`  header to `request.response` because it ensures that it's a JSON page. In the case of an error Inertia returns HTML and `JSON.parse()` fails.

Reply:
You have t check both lower case and title case variants. Depending on Inertia version, you may not have the same.

Reply:
Ok every headers are lowercased now!